src/main/java/org/junit/internal/runners/statements/RunBefores.java

```diff
@@ -20,6 +31,9 @@ public RunBefores(Statement next, List<FrameworkMethod> befores, Object target)
 
     @Override
     public void evaluate() throws Throwable {
+        if (null != previous) {
+            previous.evaluate();
```

Comment:
This seem to be unrelated to the original intent of the `RunBefores` class. If you needed a previous statement to run, couldn't you wrap the `RunBefores` with another Statement that ran `previous` and then called the next `Statement` on the chain`? Perhaps create an abstract `DelegatingStatement` method if that makes sense.